src/TextArea/TextArea.doc.js

```diff
@@ -170,3 +170,26 @@ card(
     )}
   />
 );
+
+card(
+  'Autofocus',
+  md`
+    \`TextArea\` intentionally lacks support for autofocus. Generally speaking,
+    autofocus interrupts normal page flow for screen readers making it an
+    anti-pattern for accessibility.
+  `
+);
+
+card(
+  'onSubmit',
+  md`
+    \`TextArea\` is commonly used as an input in forms along side submit buttons.
+    In these cases, users will expect the pressing Enter or Return with the input
```

Comment:
`In these cases, users will expect the pressing Enter or Return` typo here, did you mean to use `that`?

Reply:
Great catch.

PSA: Don't try to grammar at 1am.